src/chains/soroban/Soroban Report.md

```diff
@@ -161,3 +341,46 @@ Deployment scripts for Stellar Axelar components are provided, and the process i
 
 ### 4.3 Mitigation of Potential Risks
 Stellar and Axelar confirm that the system includes proper logging and mechanisms to effectively handle unusual situations in the network.
+
+---
+
+## **Section 5: Committee Members**
+
+The **Amplifier Advisory Committee** overseeing this integration consists of the following teams:
+
+- **Axelar**: Liana Spano, Coordinator
+- **Node.Monster**: Eyal Alsheich, Contributor
+- **Common Prefix**: Nikolaos Kamarinakis, Contributor; Dimitris Karakostas, Contributor
+- **Ackee**: Stepan Sonsky, Contributor
+- **Eiger**: Marcin, Contributor
+
+---
+
+## **Conclusion**
```

Comment:
Let's mention that we strongly recommend **not** proceeding with the deployment of the Stellar integration until the [Stellar Amplifier CosmWasm contracts](https://github.com/axelarnetwork/axelar-amplifier/tree/main/external-gateways/stellar) have been audited and the Ottersec audit of the [Stellar Soroban WASM edge contracts](https://github.com/axelarnetwork/axelar-amplifier-stellar) has been published, and its fixes have been confirmed resolved.